inc/modules/export/class-exporthelpers.php

```diff
@@ -49,6 +49,11 @@ public function mapBookDataAndContent( array $post_data, array $metadata, int $p
 		$data['subclass'] = $this->getPostSubClass( $post_type_identifier, $post_data['ID'] );
 		$data['slug'] = $is_epub ? $post_data['post_name'] : "{$data['post_type_class']}-{$post_data['post_name']}";
 		$data['title'] = get_post_meta( $post_data['ID'], 'pb_show_title', true ) ? $post_data['post_title'] : '';
+
+		if ( ! $is_epub ) {
+			$data['title'] = empty( $data['title'] ) ? '<span class="display-none">' . $post_data['post_title'] . '</span>' : $data['title'];
+		}
```

Comment:
Added this back because digital PDF bookmarks will display an empty bookmark if the H1 has no content.